src/Book.java

```diff
@@ -0,0 +1,72 @@
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Book {
+    private final String bookName;
+    private final String file;
+
+    public Book(String bookName) {
+        this.bookName = bookName;
+        String filePath = "D:/IT Hillel/JavaPro/lesson14/src";
+        this.file = filePath + "/" + bookName;
+    }
+
+    public String getBookName() {
+        return bookName;
+    }
+    public String getFile() {
+        return file;
+    }
+
+    public String mostPopular (FileReader bookName) throws IOException {
+        ArrayList<String> words = new ArrayList<>();
+        HashMap<String, Integer> mostPopular = new HashMap<>();
+        String text;
+        int c;
+        StringBuilder wordBuilder = new StringBuilder();
```

Comment:
not very descriptive name

Reply:
how come? as for me, its fully describe what it need for